includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -910,12 +910,6 @@ public function process_payment_for_order( $cart, $payment_information, $additio
 			$charge_id     = $intent->get_charge_id();
 			$client_secret = $intent->get_client_secret();
 			$currency      = $intent->get_currency();
-
-			if ( 'requires_action' === $status &&
-				$payment_information->is_merchant_initiated() ) {
-				// Allow 3rd-party to trigger some action if needed.
-				do_action( 'woocommerce_woocommerce_payments_payment_requires_action', $order, $intent_id, $payment_method, $customer_id, $charge_id, $currency );
-			}
```

Comment:
Why was this removed? It seems unrelated to the eslint change that this PR focuses on.

Reply:
Nice catch @mattsherman, I am not a 100% sure why I had removed that, I will revert that change.